app/lib/actions/katello/content_view_puppet_environment/create.rb

```diff
@@ -23,7 +21,7 @@ def plan(puppet_environment, clone = false)
 
             # when creating a clone, the following actions are handled by the
             # publish/promote process
-            unless clone
+            unless clone && !puppet_path
```

Comment:
I dislike compound unless, mind changing this to an `if` statement for easier readability? Now that its compound I don't think linting will yell at you.